airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/kotlin/io/airbyte/cdk/integrations/standardtest/source/PythonSourceAcceptanceTest.kt

```diff
@@ -155,7 +155,7 @@ class PythonSourceAcceptanceTest : SourceAcceptanceTest() {
         private val LOGGER: Logger = LoggerFactory.getLogger(PythonSourceAcceptanceTest::class.java)
         private const val OUTPUT_FILENAME = "output.json"
 
-        lateinit var IMAGE_NAME: String
+        var IMAGE_NAME: String = "dummy_image_name"
```

Comment:
I was going to post a comment about how this is bad but then I noticed that this class was `PythonSourceAcceptanceTest` which assuredly is dead code.